src/main/groovy/com/github/mamohr/gradle/deploymentstructure/model/Module.groovy

```diff
@@ -22,6 +22,8 @@ package com.github.mamohr.gradle.deploymentstructure.model
 class Module implements XmlSerializable {
     String name;
     String slot = 'main';
+    Set<String> excludedPaths = [];
+    Set<String> includedPaths = [];
 
```

Comment:
I think the includes and excludes would better be placed in the DependencyModule class. If placed in the module class the excludes could also get these properties, which is not allowed according to the xsd. I have to say that the Module class should actually better be called ModuleIdentifier and the DependencyModule should compose the ModuleIdentifier and not inherit from it, but that would be a refactoring todo for me.

Perhaps it would even be better to have another class FilterConfig to have includes and excludes. As the dependency module can not only have imports, but also exports a DSL like this might be more suitable. Both parts could use the a FilterConfig class

```
dependency 'my-other-dependency' {
  slot = '1.1'
  export = true
  optional = true
  annotations = true
  imports {
    include '/**'
    exclude '/abc/*'            
  }
  exports {
    exlude '/bla/*'
  }
}
```